src/liboslexec/serialize-bc.py

```diff
@@ -1,16 +1,23 @@
 # Turn an LLVM-compiled bitfile into a C++ source file where the compiled
 # bitcode is in a huge array.
 
+from __future__ import print_function, absolute_import
+
 import sys
+
 in_name = sys.argv[1]
 out_name = sys.argv[2]
 prefix = sys.argv[3]
 f_in = open(in_name, 'rb')
 f_out = open(out_name, 'w')
 f_out.write('#include <cstddef>\n')
-f_out.write('unsigned char %s_block[] = {\n' % prefix)
+f_out.write('unsigned char ' + prefix + '_block[] = {\n')
 f_in.read
-for c in f_in.read():
-    f_out.write('0x%s,\n' % c.encode('hex'))
+if (sys.version_info > (3, 0)):
+    for c in f_in.read():
+        f_out.write(hex(c) + '\n')
```

Comment:
You missed the comma after each value

f_out.write(hex(c) + ',\n')

Reply:
thanks for the tip. updated.